### ISSUE TITLE
Crash when adding a card

### ISSUE THREAD

**Issue by odedharth** [stripe/stripe-ios#999]
## Summary
There is a fatal exception that is crashing the app for some of our users when adding a credit card.

The error is:
```
Fatal Exception: NSInternalInconsistencyException
attempt to insert row 0 into section 1, but there are only 1 sections after the update
-[STPAddCardViewController addressViewModel:addedCellAtIndex:]
```
Happening at:
```
STPAddCardViewController.m line 362-[STPAddCardViewController addressViewModel:addedCellAtIndex:]
```

## Code to reproduce
It's happening for some of our users in production, and we don't know how to reproduce but we do have the stack trace of the crashes using Crashalytics: https://fabric.io/oded-harths-projects/ios/apps/oded-harth.mdacne/issues/5b1482136007d59fcdd040b0?time=last-ninety-days

## iOS version
iOS 11

## Installation method
Cocoapods

## SDK version
11.5.0

**Comment by odedharth** [comment]
@cale-stripe Sorry send the wrong link: http://crashes.to/s/90e272acb88

**Comment by odedharth** [comment]
@csabol-stripe Any ETA for the fix?

**Comment by csabol-stripe** [comment]
Fix will go into our next SDK release which is our priority at the moment! :)

**Comment by csabol-stripe** [comment]
We've just released v13.2.0 that fixes this crash. Thanks for the report and let us know if you run into any other issues!
